### PR TITLE
aria updates for openshift details with exports

### PR DESCRIPTION
**Pull request by gitdallas**
https://issues.redhat.com/browse/COST-3026

Went from
TOTAL ISSUES

to 
image

Remaining issues are either:

- PF table not having a heading for checkbox/actions
- Various insights stuff (same as last pr)
- The "review" on contrast for visibility (all looks good)

I also fixed an issue where widgetIds for paginations were incorrectly formatted and not getting parsed.

### CODE REVIEW

File: src/routes/costModels/costModel/priceListTable.tsx [hunk 1]
```diff
@@ -268,6 +268,9 @@ class PriceListTable extends React.Component<Props, State> {
                         })
                       }
                       onPerPageSelect={(_evt, perPage) => this.setState({ pagination: { page: 1, perPage } })}
+                      titles={{
+                        paginationTitle: `pricelist top pagination`,
```
Comment (dlabrecq) [inline src/routes/costModels/costModel/priceListTable.tsx:272]: String should be added to the messages file.

File: src/routes/costModels/costModel/priceListTable.tsx [hunk 2]
```diff
@@ -360,6 +363,9 @@ class PriceListTable extends React.Component<Props, State> {
                                 pagination: { page: 1, perPage },
                               })
                             }
+                            titles={{
+                              paginationTitle: `pricelist bottom pagination`,
```
Comment (dlabrecq) [inline src/routes/costModels/costModel/priceListTable.tsx:367]: String should be added to the messages file.

File: src/routes/costModels/costModelsDetails/bottomPagination.tsx [hunk 3]
```diff
@@ -34,6 +34,9 @@ function BottomPaginationBase(props: Omit<PaginationProps, 'ref'>): JSX.Element
             itemCount={itemCount}
             page={page}
             perPage={perPage}
+            titles={{
+              paginationTitle: `bottom pagination`,
```
Comment (dlabrecq) [inline src/routes/costModels/costModelsDetails/bottomPagination.tsx:38]: String should be added to the messages file.

File: src/routes/views/components/dataToolbar/dataToolbar.tsx [hunk 4]
```diff
@@ -866,7 +866,7 @@ export class DataToolbarBase extends React.Component<DataToolbarProps> {
     // Todo: clearAllFilters workaround https://github.com/patternfly/patternfly-react/issues/4222
     return (
       <div style={style ? style : styles.toolbarContainer}>
-        <Toolbar id="details-toolbar" clearAllFilters={this.onDelete as any} collapseListedFiltersBreakpoint="xl">
+        <Toolbar clearAllFilters={this.onDelete as any} collapseListedFiltersBreakpoint="xl">
```
Comment (dlabrecq) [inline src/routes/views/components/dataToolbar/dataToolbar.tsx:869]: Why has the ID been removed?

The PatternFly examples show an ID for the Toolbar component.
http://patternfly-react-main.surge.sh/components/toolbar#examples

**Review by dlabrecq — CHANGES_REQUESTED** [review]

File: src/routes/views/components/dataToolbar/dataToolbar.tsx (hunk 4 above)
Comment (gitdallas) [reply src/routes/views/components/dataToolbar/dataToolbar.tsx:869]: There was nothing referencing it, and it was being used twice within a page

**Review by dlabrecq — APPROVED** [review]
LGTM

**Pull request merged**